enhancements/cluster-logging/hypershift-audit-logs.md

```diff
@@ -21,174 +21,124 @@ see-also:
 ## Summary
 
 In a self contained cluster, users can forward application, infrastructure and audit logs.
-In a hypershift hosted cluster, API server audit logs are not exposed in the hosted cluster or on the data plane,
-they are generated in Hosted Control Plane (HCP) pods on a management cluster.
+In a hypershift hosted cluster, API server audit logs are not exposed in the hosted cluster or on the data plane.
+They are generated in Hosted Control Plane (HCP) pods on a management cluster and forwarded via HTTP web-hook.
 
 This proposal deploys the existing Cluster Logging Operator (CLO) on management and hosted clusters.
-The CLO supports both the cluster service provider and cluster service consumer personas by
-permitting destinations to be configured on the hosted control plane (for service provider) and
-the data plane (for service consumer) independently.  Each destination may independently filter
-events forwarded to each destination by way of an optional Policy.
+The CLO provides a ClusterLogForwarder resource (CLF) that collects and forwards logs.
+On the hosted cluster, a CLF can be configured to receive audit logs from the management cluster
+and forward them to customer-configured destinations.
 
-This proposal adds a "Hypershift Logging Operator" (HLO) on the management cluster.
-The HLO combines data-plane and control-plane configuration on the control plane,
-so that the CLO on the _management_ cluster can forward API audit logs for both personas.
-The CLO on the _hosted_ cluster continues to forward other logs for the consumer persona as before.
+On the management cluster, this proposal adds a "Hypershift Logging Operator" (HLO).
+The HLO watches _data-plane_ CLF resources, when it sees one configured to receive audit logs it updates a _control_plane_ CLF to do the forwarding.
```

Comment:
We were using the HLF custom resource for HLO in the initial implementation. Does this mean we are going to deprecate that and use HLO to watch the CLF directly?

Reply:
ok, seems this answered below.